etc/docker/scripts/voila_wrapper.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash
+#!/bin/bash --login
+set +euo pipefail
+conda activate voilaenv
 set -euo pipefail
```

Comment:
Did you mean to move it rather than copy it?

Reply:
Lines 2 and 4 are different. Apparently, I need to turn off strict mode to activate conda. Then I let Sergey's script take over with no changes.